.github/workflows/python-package.yml

```diff
@@ -26,12 +26,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        python -m pip install setuptools wheel
         python -m pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Install package
       run: |
-        python -m pip install -e .
-        python setup.py build
+        USE_FORTRAN=1 python -m pip install -e .
```

Comment:
We cannot use `--global-options` or `--install-options` with command line arguments for `setup.py` as it triggers a rebuild of numpy. Instead, I added an environment variable. See https://stackoverflow.com/questions/47210058/passing-command-line-arguments-to-pip-install/58491619